word2vec_decade_experiment/03_word_decade_figure_generator_2000_2020.py

```diff
@@ -0,0 +1,315 @@
+)
+
+global_distance
+
+local_distance
+
+g = plot_token_timeline(token_timeline_low_dim_df)
+g.save("output/figures/interleukin18_time_plot.png")
+print(g)
+
+plot_wordcloud_neighbors_gif(
+    token_timeline_low_dim_df, plot_filename="output/wordcloud_plots/interleukin-18.gif"
+)
+
+# ![interleukin-18 gif here](output/wordcloud_plots/interleukin-18.gif)
+
+plot_wordcloud_neighbors(
+    token_timeline_low_dim_df, plot_filename="output/wordcloud_plots/interleukin_18.png"
+)
+
+# ## p53
+
+token_timeline_df = generate_timeline(year_comparison_dict, "p53")
+token_timeline_df.head()
+
+token_timeline_low_dim_df = project_token_timeline(
+    "p53", aligned_models, model, neighbors=25
+)
+token_timeline_low_dim_df.head()
+
+global_distance, local_distance = plot_local_global_distances(
+    token_timeline_df, token="p53"
+)
+
+global_distance
+
+local_distance
+
+g = plot_token_timeline(token_timeline_low_dim_df)
+g.save("output/figures/p53_time_plot.png")
+print(g)
+
+plot_wordcloud_neighbors_gif(
+    token_timeline_low_dim_df, plot_filename="output/wordcloud_plots/p53.gif"
+)
+
+# ![p53 gif here](output/wordcloud_plots/p53.gif)
+
+plot_wordcloud_neighbors(
+    token_timeline_low_dim_df, plot_filename="output/wordcloud_plots/p53.png"
+)
+
+# ## Cystic
+
+token_timeline_df = generate_timeline(year_comparison_dict, "cystic")
+token_timeline_df.head()
+
+token_timeline_low_dim_df = project_token_timeline(
+    "cystic", aligned_models, model, neighbors=25
+)
+token_timeline_low_dim_df.head()
+
+global_distance, local_distance = plot_local_global_distances(
+    token_timeline_df, token="cystic"
+)
+
+global_distance
+
+local_distance
+
+g = plot_token_timeline(token_timeline_low_dim_df)
+g.save("output/figures/cystic_time_plot.png")
+print(g)
+
+plot_wordcloud_neighbors_gif(
+    token_timeline_low_dim_df, plot_filename="output/wordcloud_plots/cystic.gif"
+)
+
+# ![cystic_gif here](output/wordcloud_plots/cystic.gif)
+
+plot_wordcloud_neighbors(
+    token_timeline_low_dim_df, plot_filename="output/wordcloud_plots/cystic.png"
+)
+
+# ## Cell
+
+token_timeline_df = generate_timeline(year_comparison_dict, "cell")
+token_timeline_df.head()
+
+token_timeline_low_dim_df = project_token_timeline(
+    "cell", aligned_models, model, neighbors=25
+)
+token_timeline_low_dim_df.head()
+
+global_distance, local_distance = plot_local_global_distances(
+    token_timeline_df, token="cell"
+)
+
+global_distance
+
+local_distance
+
+g = plot_token_timeline(token_timeline_low_dim_df)
+g.save("output/figures/celltime_plot.png")
+print(g)
+
+plot_wordcloud_neighbors_gif(
+    token_timeline_low_dim_df, plot_filename="output/wordcloud_plots/cell.gif"
+)
+
+# ![cell gif here](output/wordcloud_plots/cell.gif)
+
+plot_wordcloud_neighbors(
+    token_timeline_low_dim_df, plot_filename="output/wordcloud_plots/cell.png"
+)
+
+# # Conclusions - Take Home Point(s)
+
+# 1. 2008 - 2010 has a huge spike in semantic change for all the tokens I have analyzed in this notebook
```

Comment:
Why is that happening? Can you give an explanation for this? Did something happen maybe?

Reply:
>Why is that happening? Can you give an explanation for this? Did something happen maybe?

Right that is a good question. Part of me wants to say that's when there was a spike in open access abstracts, but no idea. Will need to dig deeper into this trend after this PR to know more.